ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -4353,6 +4353,17 @@ tests:
     - ref: cucushift-installer-check-realtime
     - chain: openshift-e2e-test-qe
     workflow: baremetal-lab-upi-dual-stack
+- as: aws-short-cert-rotation-f9
+  cron: 15 1 9,19,28 * *
```

Comment:
The cron job should generate by the ./ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh
i.e.,
./ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh aws-ipi-ovn-hypershift-guest-longduration-f14 openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml 
cron: 13 18 4,18 * *